keps/sig-storage/1682-csi-driver-skip-permission/README.md

```diff
@@ -182,22 +179,19 @@ A test plan will include the following tests:
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
-
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g. what if some components will restart
-  in the middle of rollout?
+  No, a rollout should not impact running workloads, since the default behavior
+  remains the same and a pod spec change is required to use the new field.
 
 * **What specific metrics should inform a rollback?**
+  No known rollback criteria.
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
   Describe manual testing that was done and the outcomes.
```

Comment:
@huffmanca @gnufied Do either of you have any details on upgrade / rollback testing that could be added here?

Reply:
upgrade->downgrade->upgrade path should work fine IMO, since the entire behaviour is driver by `CSIDriver` object - so if the field is not present it defaults to current heuristic of looking at fstype and RWO volumes for applying fsgroups.